HousingFinanceInterimApi/Handler.cs

```diff
@@ -127,8 +139,28 @@ public Handler()
                 reportGateway, transactionGateway, googleFileSettingGateway, googleClientService);
             _moveHousingBenefitFileUseCase = new MoveHousingBenefitFileUseCase(batchLogGateway, batchLogErrorGateway,
                 googleFileSettingGateway, googleClientService);
+            _updateTAUseCase = new UpdateTAUseCase(updateTAGateway);
+
         }
 
+        public async Task DynamodbStream(DynamoDBEvent dynamoDBEvent)
+        {
+            var request = new UpdateTARequest();
+            foreach (var record in dynamoDBEvent.Records)
```

Comment:
Is the foreach-loop needed?
What is the size of the Records collection used in this loop? 

Is it not possible to get the exact `tag_ref` that is being updated from the `dynamoDBEvent` argument?

Reply:
The for-each is required as there could be more than one record updated at the same time - within the DynamodbEvent.Records there would only be the records that have been updated and it only takes seconds to update the finance databases so i don't think having a loop would be an issue

Reply:
But from the MMH form we are only updating 1 record at a time and not multiple records. 

What is the size of the `dynamoDBEvent.Records` collection used in this loop?